src/System/Wlog/Formatter.hs

```diff
@@ -169,23 +169,28 @@ centiUtcTimeF t =
   where
     centiSecondF = padRightF 3 '0' . T.take 3 . fmt . subsecondF
 
+-- | Get the current time rounded to 10^n picoseconds.
```

Comment:
I think you also need to change documentation for `HandlerWrap` data type:

https://github.com/serokell/log-warper/blob/7bc6f5446fd411b457864455ab53be3a77a4ae19/src/System/Wlog/LoggerConfig.hs#L117

Reply:
Done